wolox-books/src/app/app.component.html

```diff
@@ -1,21 +1,18 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
-  <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
+<div class="navbar">
+  <img class="logo" src="../assets/wolox_books.png" alt="WOLOX">
+</div>
+<div class="main">
+  <form class="auth-form" [formGroup]="form" (ngSubmit)="register()">
+    <label class="bold prefix" for="first_name">First name</label>
+    <input class="data" id="first_name" type="text" formControlName="first_name">
+    <label class="bold prefix" for="last_name">Last name</label>
+    <input class="data" id="last_name" type="text" formControlName="last_name">
+    <label class="bold prefix" for="email">Email</label>
+    <input class="data" id="email" type="text" formControlName="email">
+    <label class="bold prefix" for="password">Password</label>
+    <input class="data" id="password" type="password" formControlName="password">
+    <input type="submit" class="button main-button" value="Sign up" [disabled]="!form.valid">
```

Comment:
for submit buttons we use `<button type="submit"></button>` instead of input type submit since the value should be placed as content and not as attribute.